src/main/flight/pid.c

```diff
@@ -183,7 +183,26 @@ void resetPidProfile(pidProfile_t *pidProfile)
                  .crash_gthreshold = 400,                  // degrees/second
                  .crash_setpoint_threshold = 350,          // degrees/second
                  .crash_recovery = PID_CRASH_RECOVERY_OFF, // off by default
-                 .horizon_tilt_effect = 130, .nfe_racermode = false, .cinematic_setpoint = false, .crash_limit_yaw = 200, .itermLimit = 400, .throttle_boost = 5, .throttle_boost_cutoff = 15, .iterm_rotation = true, .iterm_relax = ITERM_RELAX_OFF, .iterm_relax_cutoff = 11, .iterm_relax_type = ITERM_RELAX_GYRO, .abs_control_gain = 0, .abs_control_limit = 90, .abs_control_error_limit = 20, .antiGravityMode = ANTI_GRAVITY_SMOOTH, .use_integrated_yaw = false, .integrated_yaw_relax = 200, .motor_output_limit = 100, .auto_profile_cell_count = AUTO_PROFILE_CELL_COUNT_STAY, .horizonTransition = 0, );
+                 .horizon_tilt_effect = 130,
```

Comment:
👍

Reply:
ya with the recent changes to make pid.c nicer I can't believe we overlooked that lol.